src/Core/Checkout/Cart/Event/BeforeLineItemQuantityChangedEvent.php

```diff
@@ -56,4 +58,14 @@ public function getSalesChannelContext(): SalesChannelContext
     {
         return $this->salesChannelContext;
     }
+
+    public function getBeforeUpdateQuantity(): int
+    {
+        return $this->beforeUpdateQuantity;
+    }
+
+    public function setBeforeUpdateQuantity(int $beforeUpdateQuantity): void
```

Comment:
is there a reason, why you use a setter instead of a parameter in the constructor?

Reply:
should we make it mandatory for 6.7?

 @Bird87ZA would you take care?

Reply:
Yes. :-)